firmware/examples/Gpio/source/main.cpp

```diff
@@ -0,0 +1,33 @@
+#include "L1_Drivers/gpio.hpp"
+
+int main(void)
+{
+    // Port 2, pin 0 must be connected to a switch that is able to ground and
+    // supply 3.3 volts to the pin (You can use a 1kOhm resistor to connect the
+    // pin to the switch for protection).
+    // Port 2, pin 2 must be connected to an LED that connects to ground (use
+    // the appropriate resistor to protect the LED).
+    Gpio p2_00(2, 0);
+    Gpio p2_02(2, 2);
+
+    DEBUG_PRINT("Configure port 2, pin 0 as input and port 2, pin 2 as output");
```

Comment:
If you see the other examples. They typically start with, "Starting ____ application..." Can you follow that scheme.
Each debug print should be its own thing, so you before you set P2.00 to input, you shoudl have a message and another for setting p2.02 to output.